### PR TITLE
Add weekly updates 2015/07/10

### PR DESCRIPTION
**Pull request by yosuke-furukawa**

### CODE REVIEW

**Comment by yosuke-furukawa** [comment]
please review my PR!
@nodejs/evangelism

File: weekly-updates/weekly-update.2015-07-10.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+### io.js and Node.js News — July 10th
+security patches for io.js 1.8 and 2.3 and upcoming events.
```
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-07-10.md]: "Security" needs to be capitalized.

**Comment by diagramatics** [comment]
Apart from that, LGTM.

**Comment by yosuke-furukawa** [comment]
Thanks! I will merge my PR when another `LGTM` is coming.

**Comment by yosuke-furukawa** [comment]
We should avoid the weekly update is delayed one week...
I will publish this before next weekly update will begin.

**Pull request merged**